package.json

```diff
@@ -110,7 +114,7 @@
     "deep-freeze": "^0.0.1",
     "dotenv": "^8.0.0",
     "errorhandler": "^1.5.1",
-    "express": "^4.17.1",
+    "express": "4.17.0",
```

Comment:
Is there a reason we are not using compatible patch versions here?

Reply:
yes, the Apollo server library pulls its own express dependency. Want to avoid conflicts due to that.